fbw-a380x/src/systems/systems-host/systems/tcas/components/LegacyTcasComputer.ts

```diff
@@ -0,0 +1,1336 @@
+      this.activeRa.hasBeenAnnounced = this._newRa.hasBeenAnnounced;
+      this.activeRa.secondsSinceStart += _deltaTime / 1000;
+      if (!this.activeRa.hasBeenAnnounced) {
+        if (this.debug) {
+          console.log('RA Intruders: ');
+          console.log(' ================================ ');
+          this.raTraffic.forEach((traffic) => {
+            console.log(` id | ${traffic.ID}`);
+            console.log(` alt | ${traffic.alt}`);
+            console.log(` rAlt | ${traffic.relativeAlt}`);
+            console.log(` sDist | ${traffic.slantDistance}`);
+            console.log(
+              ` bearing | ${MathUtils.computeGreatCircleHeading(this.ppos.lat, this.ppos.long, traffic.lat, traffic.lon)}`,
+            );
+            console.log(
+              ` hDist | ${MathUtils.computeGreatCircleDistance(this.ppos.lat, this.ppos.long, traffic.lat, traffic.lon)}`,
+            );
+            console.log(` closureRate | ${traffic.closureRate}`);
+            console.log(` closureAccel | ${traffic.closureAccel}`);
+            console.log(` RA TAU | ${traffic.raTau} <<< ${TCAS.TAU[this.sensitivity.getVar()][TaRaIndex.RA]}`);
+            console.log(` V TAU | ${traffic.vTau} <<< ${TCAS.TAU[this.sensitivity.getVar()][TaRaIndex.TA]}`);
+            console.log(` TA TAU | ${traffic.taTau} <<< ${TCAS.TAU[this.sensitivity.getVar()][TaRaIndex.RA]}`);
+            console.log(' ================================ ');
+          });
+          console.log('TCAS: RA GENERATED: ', this.activeRa.info.callout);
+        }
+
+        if (this.activeRa.info.callout.repeat) {
+          this.soundManager.tryPlaySound(this.activeRa.info.callout.sound, true, true);
+        } else {
+          this.soundManager.tryPlaySound(this.activeRa.info.callout.sound, true, false);
+        }
+
+        const isCorrective = this.activeRa.info.type === RaType.CORRECT;
+        this.correctiveRa.setVar(isCorrective);
+        this.raType.setVar(this.activeRa.info.type2);
+        this.rateToMaintain.setVar(this.activeRa.info.rateToMaintain);
+        this.upAdvisoryStatus.setVar(this.activeRa.info.upAdvisory);
+        this.downAdvisoryStatus.setVar(this.activeRa.info.downAdvisory);
+        SimVar.SetSimVarValue('L:A32NX_TCAS_VSPEED_RED:1', 'Number', this.activeRa.info.vs.red[Limits.MIN]);
+        SimVar.SetSimVarValue('L:A32NX_TCAS_VSPEED_RED:2', 'Number', this.activeRa.info.vs.red[Limits.MAX]);
+        if (isCorrective) {
+          SimVar.SetSimVarValue('L:A32NX_TCAS_VSPEED_GREEN:1', 'Number', this.activeRa.info.vs.green[Limits.MIN]);
+          SimVar.SetSimVarValue('L:A32NX_TCAS_VSPEED_GREEN:2', 'Number', this.activeRa.info.vs.green[Limits.MAX]);
+        }
+
+        this.activeRa.hasBeenAnnounced = true;
+      }
+    }
+  }
+
+  /**
+   * Send intruder array to ND display
+   */
+  private emitDisplay(): void {
+    this.sendAirTraffic.length = 0;
+    const sentAirTraffic = this.airTraffic
```

Comment:
You can if (efisTrafFilterOn) around this block, excepting the last line to ensure you do actually send the empty array.

Reply:
Have to split into L/R as well I think, but its one idea.

Reply:
Yeah, I guess for that it needs to go on the ND side.